docs/modules/ROOT/pages/how-to-guides/testing_applications/proc_adding_an_integration_test.adoc

```diff
@@ -22,6 +22,8 @@ Complete the following steps in the {ProductName} console:
 .. For components using the default build pipeline, go to the *Components* tab, select the three dots next to the name of the component, and select *Start new build*.
 .. For components with an upgraded build pipeline, make a commit to their GitHub repository.
 
+WARNING: Without specifying *Environment* an integration test can do only static testing because application won't be deployed into an ephemeral environment and application endpoints won't be available for testing.
```

Comment:
Thanks for the PR. I appreciate the need to emphasize this point. However, could you please move this warning up to line 19, so it immediately follows the relevant point (#7)? And could you please adjust the wording as follows: "WARNING: Without specifying an *Environment*, an integration test can only do static testing. To make application endpoints available for testing, you must deploy your application to an ephemeral environment."

Reply:
updated